packages/components/src/components/CodeEditor.tsx

```diff
@@ -243,6 +261,38 @@ export const CodeEditor = forwardRef<CodeEditorHandle, CodeEditorProps>(
       });
     }, [format]);
 
+    useEffect(() => {
+      editorView.current?.dispatch({
+        effects: compViewNodeListener.reconfigure(
+          keymap.of([
+            {
+              key: "Alt-Shift-v",
```

Comment:
I think we should later come up with a different key combo.

Maybe we could also have a button in the menu that does this? Ideally it could only appear when there's a relevant variable the user selects.

Reply:
I plan to experiment with context menus later.

On keys: Alt+Shift+v is parallel to Alt+Shift+f for format. Alt+v might also be possible; Anything with Cmd would risk collisions with system/browser bindings (though I'm considering Cmd+s that would hijack rarely used "Save page" binding for saving squigglehub models).